docs/reference/migration/migrate_7_0/settings.asciidoc

```diff
@@ -2,6 +2,12 @@
 
 === Settings changes
 
+==== The default for `node.name` is now the hostname
+
+`node.name` now defaults to the hostname at the time when Elasticsearch
+is started. We hope that this is a more generally useful default than
```

Comment:
I would replace the second sentence with:

> Previously the default node name was the first eight characters of the node id. It can still be configured explicitly in `elasticsearch.yml`.